PerspectiveBoxes/subdivide_quad.test.js

```diff
@@ -0,0 +1,100 @@
+import { describe, it, expect } from "vitest";
+import { Point } from "../pga2d/objects";
+import { PerspectiveQuad } from "./PerspectiveQuad";
+import { subdivide_quad } from "./subdivide_quad";
+import { PGA_TESTERS } from "../pga2d/pga_testers";
+
+//expect.addEqualityTesters(PGA_TESTERS);
```

Comment:
If I can't get the testers working, I guess I'll need to make an array equals matcher. kinda annoying.